deploy-manage/deploy/cloud-enterprise/enable-custom-endpoint-aliases.md

```diff
@@ -23,21 +23,21 @@ After installing or upgrading to version 2.10 or later:
 2. [Update your proxy certificate(s)](../../security/secure-your-elastic-cloud-enterprise-installation/manage-security-certificates.md). In addition to currently configured domains, additional SAN entries must be configured for each application-specific subdomain:
 
     ::::{note}
-    If you are not using wildcard certificates, you need to repeat this process for each deployment to account for specific aliases.
+    If you are not using wildcard certificates, you need to repeat this process for each deployment to account for specific aliases. Review [Wildcard DNS record and certificates](./ece-wildcard-dns.md) for more guidance.
     ::::
-
+    
 
     * For {{es}}, the certificate needs to allow for **\*.es.<your-domain>**
     * For {{kib}}, the certificate needs to allow for **\*.kb.<your-domain>**
     * For APM, the certificate needs to allow for **\*.apm.<your-domain>**
     * For Fleet, the certificate needs to allow for **\*.fleet.<your-domain>**
     * For Universal Profiling, the certificate needs to allow for **\*.profiling.<your-domain>** and **\*.symbols.<your-domain>**
 
+    ::::{note} 
+    If you plan to deploy [Integration Servers](../../deploy/cloud-enterprise/manage-integrations-server.md), you must add two additional wildcard subdomains, `*.fleet.<your-domain>` and `*.apm.<your-domain>`, to the Subject Alternative Names (SANs) attached to the proxy wildcard certificate.
+    ::::
```

Comment:
```suggestion
```

I'm suggesting removing the note, as Fleet and APM appear in the previous list.

Reply:
Yeah great idea! I didn't notice we use fleet and apm to represent integrations server, and it's already there. Thanks for pointing this out!